sql/moz-fx-data-shared-prod/telemetry_derived/accessibility_clients_v1/query.sql

```diff
@@ -0,0 +1,141 @@
+  WHERE
+    submission_date = @submission_date
+    AND (
+      (histogram_parent_a11y_consumers IS NOT NULL AND hpac.value = 1)
+      OR SPLIT(scalar_parent_a11y_instantiators, '|')[OFFSET(0)] IN (
+        'Magnify.exe',
+        'Narrator.exe',
+        'jfw.exe',
+        'Zt.exe',
+        "nvda.exe",
+        "osk.exe",
+        "sapisvr.exe",
+        'TabTip.exe',
+        "VoiceOver"
+      )
+    )
+)
+SELECT
+  submission_date,
+  normalized_channel,
+  COUNT(
+    DISTINCT
+    CASE
+    WHEN
+      consumer = 0
+      OR instantiator = 'nvda.exe'
+    THEN
+      client_id
+    ELSE
+      NULL
+    END
+  ) AS NVDA,
+  COUNT(
+    DISTINCT
+    CASE
+    WHEN
+      consumer = 1
+      OR instantiator = 'jfw.exe'
+    THEN
+      client_id
+    ELSE
+      NULL
+    END
+  ) AS JAWS,
+  COUNT(DISTINCT CASE WHEN consumer = 2 THEN client_id ELSE NULL END) AS OLDJAWS,
+  COUNT(
+    DISTINCT
+    CASE
+    WHEN
+      consumer = 3
+      OR UPPER(instantiator) LIKE UPPER('%WindowEyes%')
+    THEN
+      client_id
+    ELSE
+      NULL
+    END
+  ) AS WE,
+  COUNT(
+    DISTINCT
+    CASE
+    WHEN
+      consumer = 4
+      OR instantiator = 'Snova.exe'
+    THEN
+      client_id
+    ELSE
+      NULL
+    END
+  ) AS DOLPHIN,
+  COUNT(DISTINCT CASE WHEN consumer = 5 THEN client_id ELSE NULL END) AS SEROTEK,
+  COUNT(DISTINCT CASE WHEN consumer = 6 THEN client_id ELSE NULL END) AS COBRA,
+  COUNT(
+    DISTINCT
+    CASE
+    WHEN
+      consumer = 7
+      OR instantiator = 'Zt.exe'
+    THEN
+      client_id
+    ELSE
+      NULL
+    END
+  ) AS ZOOMTEXT,
+  COUNT(DISTINCT CASE WHEN consumer = 8 THEN client_id ELSE NULL END) AS KAZAGURU,
+  COUNT(DISTINCT CASE WHEN consumer = 9 THEN client_id ELSE NULL END) AS YOUDAO,
+  COUNT(DISTINCT CASE WHEN consumer = 10 THEN client_id ELSE NULL END) AS UNKNOWN,
+  COUNT(DISTINCT CASE WHEN consumer = 11 THEN client_id ELSE NULL END) AS UIAUTOMATION,
+  COUNT(DISTINCT CASE WHEN consumer = 12 THEN client_id ELSE NULL END) AS VISPEROSHARED,
+  COUNT(DISTINCT CASE WHEN instantiator = 'Magnify.exe' THEN client_id ELSE NULL END) AS MAGNIFIER,
+  COUNT(DISTINCT CASE WHEN instantiator = 'Narrator.exe' THEN client_id ELSE NULL END) AS NARRATOR,
+  COUNT(
+    DISTINCT
+    CASE
+    WHEN
+      instantiator = 'osk.exe'
+    THEN
+      client_id
+    ELSE
+      NULL
+    END
+  ) AS ON_SCREEN_KEYBOARD,
+  COUNT(
+    DISTINCT
+    CASE
+    WHEN
+      instantiator = 'sapisvr.exe'
+    THEN
+      client_id
+    ELSE
+      NULL
+    END
+  ) AS SPEECH_RECONGITION,
+  COUNT(
+    DISTINCT
+    CASE
+    WHEN
+      instantiator = 'TabTip.exe'
+    THEN
+      client_id
+    ELSE
+      NULL
+    END
+  ) AS TOUCH_KEYBOARD_AND_HANDWRITING_PANEL,
+  COUNT(DISTINCT CASE WHEN instantiator = 'VoiceOver' THEN client_id ELSE NULL END) AS VOICE_OVER,
+  COUNT(DISTINCT CASE WHEN consumer NOT BETWEEN 0 AND 12 THEN client_id ELSE NULL END) AS OTHER
+FROM
+  a11y_consumers_instantiators
+GROUP BY
+  1,
+  2
```

Comment:
Preferred style is to be explicit about the fields used for grouping, 
```suggestion
  submission_date,
  normalized_channel
```